bodhi/server/tasks/approve_testing.py

```diff
@@ -80,14 +80,17 @@ def approve_update(update: Update, db: Session):
         log.info(f"{update.alias} has not met testing requirements - bailing")
         return
     log.info(f'{update.alias} now meets testing requirements')
-    # Only send email notification about the update reaching
-    # testing approval on releases composed by bodhi
-    update.comment(
-        db,
-        str(config.get('testing_approval_msg')),
-        author='bodhi',
-        email_notification=update.release.composed_by_bodhi
-    )
+    # If the update is going to be pushed automatically to stable, do not
+    # double comment that the maintainer can push it manually (#3846)
+    if not update.autotime or update.days_in_testing < update.stable_days:
```

Comment:
I understand the first part of the if statement, the second part surprises me though, especially because we were not checking this here before. Isn't it being checked earlier on? Maybe as part of "meets_testing_requirements"?

Reply:
yeah, it is quite tricky...
There can be a case where autotime is on, the update has not yet been enough time in testing, but it meets testing requirements.

The `meets_testing_requirements` evaluates to True 
```if self.karma >= self.release.critpath_min_karma```
so for current releases, an update can be manually pushed to stable after reaching +2 karma, regardless of how much time has been in testing, even if autopush is set to automatically push after +3 karma...